src/app/search/search-filters/range-overload.tsx

```diff
@@ -137,3 +172,12 @@ export default overloadedRangeFilters;
 function replaceSeasonTagWithNumber(s: string) {
   return s.replace(/[a-z]+$/i, (tag) => seasonTagToNumber[tag]);
 }
+
+/**
+ * replaces a word with a corresponding power level
+ *
+ * use only on simple filter values where there's not other letters
+ */
+function replacePowerLevelKeyword(s: string) {
+  return s.replace(new RegExp(powerLevelKeywords.join('|')), (tag) => powerLevelByKeyword[tag]);
```

Comment:
Nice! Theoretically could extract the regex to a single initialization.

Reply:
thanks